src/GovUk.Education.ExploreEducationStatistics.Notifier/local.settings.json

```diff
@@ -6,6 +6,8 @@
     "NotifyApiKey": "change-me",
     "VerificationEmailTemplateId": "change-me",
     "PublicationNotificationEmailTemplateId": "change-me",
+    "OriginalReleasePublicationEmailTemplateId": "change-me",
+    "AmendedReleasePublicationEmailTemplateId": "change-me",
```

Comment:
I think these both replace `PublicationNotificationEmailTemplateId` which needs to be removed.